single_kernel_mongo/managers/ldap.py

```diff
@@ -191,4 +194,36 @@ def get_status(self) -> StatusBase | None:
                 logger.info("Waiting for LDAP data.")
                 return BlockedStatus("Missing LDAP data from Glauth.")
             case _:
-                return get_ldap_connection_status(self.state.ldap, self.state.paths)
+                return self.get_ldap_connection_status()
+
+    def get_ldap_connection_status(self) -> StatusBase:
+        """Checks if the LDAP connection is working or not.
```

Comment:
"""Returns a status for LDAP connections

This involves checking that each LDAP uri can connect to the server